src/groupeddataframe/splitapplycombine.jl

```diff
@@ -741,74 +775,45 @@ function _combine(f::AbstractVector{<:Pair}, gd::GroupedDataFrame)
         # This can speed up some aggregates that would not trigger this on their own
         @assert gd.idx !== nothing
     end
-    res = map(f) do p
-        if isagg(p)
-            incol = gd.parent[!, first(p)]
-            agg = check_aggregate(last(p))
-            outcol = agg(incol, gd)
-            return idx_agg, outcol
-        else
-            fun = do_f(last(p))
-            if p isa Pair{<:ColumnIndex}
-                incols = gd.parent[!, first(p)]
-            else
-                df = gd.parent[!, collect(first(p))]
-                incols = NamedTuple{Tuple(names(df))}(eachcol(df))
-            end
-            firstres = do_call(fun, gd.idx, gd.starts, gd.ends, gd, incols, 1)
-            idx, outcols, _ = _combine_with_first(wrap(firstres), fun, gd, incols)
-            return idx, outcols[1]
-        end
-    end
+    res = Vector{Any}(undef, length(f))
+    for (i, p) in enumerate(f)
+           if isagg(p)
```

Comment:
Incorrect indentation.

Reply:
fixed; for some reason my editor messes up indentation sometimes (sorry for this, as you spot these problems too often :cry:)